languages/en.php

```diff
@@ -34,6 +34,9 @@
 	'event_calendar:day_label' => "Day",
 	'event_calendar:week_label' => "Week",
 	'event_calendar:month_label' => "Month",
+	'event_calendar:day_label_with_separator' => "Day: ",
+	'event_calendar:week_label_with_separator' => "Week: ",
+	'event_calendar:month_label_with_separator' => "Month: ",
```

Comment:
I guess this is a valid use case indeed. I'll look into it. Though I won't be able to work on it this week anymore.